carma-messenger-core/cpp_message/include/generated/AccelSteerYawRateConfidence.h

```diff
@@ -1,3 +1,21 @@
+
+/*------------------------------------------------------------------------------
+* Copyright (C) 2020-2021 LEIDOS.
+*
+* Licensed under the Apache License, Version 2.0 (the "License"); you may not
+* use this file except in compliance with the License. You may obtain a copy of
+* the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+* WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+* License for the specific language governing permissions and limitations under
+* the License.
+
+------------------------------------------------------------------------------*/
```

Comment:
Everything under the generated/ folder does not need to have the copyright marked. This are auto generated files and I'm not sure we really have the copyright on them in that case.